### PR TITLE
Multiple Types

### PR DESCRIPTION
**Pull request by NaominoDino**
Added multiple types to handle down (not fully implemented!)
Small fixes, headers.
Hide cursor on Window Focus (Config + Window)
(Allow edits by maintainers)
Good luck, good night!

### CODE REVIEW

**Comment by raphaelthegreat** [comment]
This is the third PR you have opened and it still has mistakes (you duplicated UNREACHABLE). Also a ton of commit messages with noise. I won't merge this until you correct all mistakes and you need to redo this

**Comment by red-prig** [comment]
As far as I know, PT_INTERP will never be present in orbis elf files, and PT_SCE_LIBVERSION, if I'm not mistaken, is typical only for PS5

File: src/video_core/renderer_vulkan/liverpool_to_vk.cpp [hunk 1]
```diff
@@ -223,6 +223,7 @@ vk::CompareOp DepthCompare(AmdGpu::DepthCompare comp) {
     case AmdGpu::DepthCompare::Always:
         return vk::CompareOp::eAlways;
     default:
+        UNREACHABLE_MSG("Unknown depth compare op {}", u32(comp));
```
Comment (ryzendew) [inline src/video_core/renderer_vulkan/liverpool_to_vk.cpp:226]: Why is this done twice? if your adding a new one the old one should be removed

Comment (NaominoDino) [reply src/video_core/renderer_vulkan/liverpool_to_vk.cpp:226]: Someone updated the branch and I just synced it.

**Comment by NaominoDino** [comment]
@red-prig It's from PS4 Minecraft version.

**Comment by red-prig** [comment]
What I said was wrong, you can just ignore PT_INTERP btw

**Pull request closed without merging**